plat/drivers/rtc/pl031.c

```diff
@@ -208,8 +213,77 @@ static const struct rtc_ops ops_pl031 = {
 	.irq_handler		= pl031_irq_handler,
 };
 
-struct rtc_dev rtc_pl031 = {
+static struct rtc_dev rtc_pl031 = {
 	.name	= "rtc_pl031",
 	.id	= 0,
 	.ops	= &ops_pl031,
 };
+
+void rtc_init_pl031(void *dtb)
+{
+	__u64 size;
+	__u32 irq_type, hwirq, trigger_type;
+	int fdt_rtc, ret, index, irq, rc;
+
+	uk_pr_info("Probing RTC...\n");
+	/*
+	 * We choose the first available rtc device in device list as the
+	 * system rtc.
+	 */
+	fdt_rtc = fdt_node_offset_idx_by_compatible_list(dtb, -1,
```

Comment:
using fdt_node_offset_by_compatible to find "arm,pl031" is better here, as the index is redundant.

Reply:
done